specs/core/1_custody-game.md

```diff
@@ -509,16 +628,28 @@ def process_challenge_deadlines(state: BeaconState) -> None:
             records[records.index(challenge)] = CustodyBitChallengeRecord()
 ```
 
+Run `clean_up_exposed_randao_key_reveals(state)` after `process_final_updates(state)`:
+
+```python
+def clean_up_exposed_randao_key_reveals(state: BeaconState) -> None:
+
+    # Clean up exposed RANDAO key reveals
+    state.exposed_randao_reveals[current_epoch % RANDAO_PENALTY_MAX_FUTURE_EPOCHS] = []
```

Comment:
Why not just make this line be part of `process_final_updates`?

Reply:
It essentially is -- I thought it would be nicer in terms of testing to have a functional separation of phase 0 and phase1 stuff. Happy to go back to the "patch" approach, but we will have to find a way to make it work with the function puller

Reply:
Done it your way for now, I don't really have a strong opinion :)